### PR TITLE
fix(version): microservice-catalog sandbox examples latest version

### PR DESCRIPTION
**Pull request by Jyoti-13**
## Description

Updated sandox examples dependencies to the latest version.

## Checklist:

- [ ]  Performed a self-review of my own code
- [x]  npm test passes on your machine
- [ ]  New tests added or existing tests modified to cover all changes
- [ ]  Code conforms with the style guide
- [ ]  API Documentation in code was updated
- [ ]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by akshatdubeysf** [comment]
@Jyoti-13 fix PR title and description

**Review by akshatdubeysf — APPROVED** [review]

File: packages/core/package-lock.json [hunk 1]
```diff
@@ -1,8965 +1,8 @@
 {
   "name": "@sourceloop/core",
   "version": "5.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (samarpan-b) [inline packages/core/package-lock.json]: revert this

**Review by samarpan-b — CHANGES_REQUESTED** [review]
revert all package locks of services

File: packages/core/package-lock.json (hunk 1 above)
Comment (Jyoti-13) [reply packages/core/package-lock.json]: Reverted.

**Comment by samarpan-b** [comment]
Please update dockerfile as well @Jyoti-13

**Comment by Jyoti-13** [comment]
> Please update dockerfile as well @Jyoti-13

Done.

File: sandbox/audit-ms-example/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Check out https://hub.docker.com/_/node to select a new base image
-FROM node:14-slim
+FROM node:16-slim
```
Comment (samarpan-b) [inline sandbox/audit-ms-example/Dockerfile:2]: use node:16-alpine

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: sandbox/audit-ms-example/Dockerfile (hunk 2 above)
Comment (Jyoti-13) [reply sandbox/audit-ms-example/Dockerfile]: updated

**Pull request merged**